test/integration/dra/dra_test.go

```diff
@@ -146,21 +170,42 @@ func TestDRA(t *testing.T) {
 			etcdOptions := framework.SharedEtcd()
 			apiServerOptions := kubeapiservertesting.NewDefaultTestServerOptions()
 			apiServerFlags := framework.DefaultTestServerFlags()
-			// Default kube-apiserver behavior, must be requested explicitly for test server.
-			runtimeConfigs := []string{"api/alpha=false", "api/beta=false"}
+			var runtimeConfigs []string
```

Comment:
Drive-by change: I had changed the defaults of the test server in a different PR.